x-pack/plugins/cases/server/connectors/case/index.ts

```diff
@@ -86,6 +86,9 @@ async function executor(
     userActionService,
     alertsService,
     logger,
+    // sub-cases-enabled: force this to true because we should never be able to call execute if the feature is disabled in the first
+    // place. The plugin won't register case as a connector if subCasesEnabled is false.
+    subCasesEnabled: true,
```

Comment:
As a precaution measure maybe we should throw an error if  `subCasesEnabled: true` so we don't allow the execution of the connector at all.

Reply:
Good idea, I added it.